.gitignore

```diff
@@ -74,3 +74,4 @@ bin/*
 
 # Most used editor files
 *.vscode
+applications/GeoMechanicsApplication/tests/test_interfaces/test_interface_side_cohesive.gid/data.dpdbg.rr0
```

Comment:
please don't add this here, but in your local gitignore if you need it

Reply:
the added line is removed.